test/simple.test.js

```diff
@@ -118,7 +118,7 @@ describe('simple', () => {
 
     it('should be indentation and "empty line before" warnings', () => {
       const warnings = data.results[0].warnings.reduce(
-        (all, { rule }) => (all.includes(rule) ? all : all.concat(rule)),
```

Comment:
If we're compiling everything correctly now, can't we use this `includes` ESNext syntax?

Reply:
@emilgoldsmith No. Babel only transform codes in `src`, but not `test`.

Reply:
Ahh yes of course, didn't think of that, true!